tcms/testruns/migrations/0007_auto_20191004_2053.py

```diff
@@ -0,0 +1,109 @@
+# Generated by Django 2.2.6 on 2019-10-04 20:53
+
+from django.db import migrations, models
+
+from tcms.testruns.models import TestExecutionStatus
```

Comment:
Don't import models in migrations. Use `apps.get_model` inside the forward/reverse methods. See testplans/migrations/0005_squashed.py for example.